pkg/workloads/tf_api/requirements.txt

```diff
@@ -2,15 +2,3 @@ flask==1.0.2
 Flask-API==1.1
```

Comment:
Do you think we should add a comment, saying that you must use `name==version` syntax with nothing else on that line (e.g. no in-line comments, no ` --hash=sha256:...`, no `>=`, `<=`, `<`, etc?